src/components/common/Card/Card.jsx

```diff
@@ -1,14 +1,15 @@
-import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+function Card({ isEmpty, cardInfo }) {
+  const { number1, number2, number3, number4, owner, month, year } = cardInfo;
 
-function Card({ isEmpty, handleCardAdd, cardInfo }) {
   return (
     <div className="card-box">
-      <div
-        className={isEmpty ? 'card-container card-empty' : 'card-container card-not-empty'}
-        onClick={handleCardAdd}
-      >
+      <div className={isEmpty ? 'card-container card-empty' : 'card-container card-not-empty'}>
```

Comment:
이번에 수정한 게 아니라 이번 PR의 리뷰 대상이 아니긴 한데, 지금은 ... 학습과정이니ㅎㅎ; (+ 이전에 놓친 저를 반성하며..)
`card-container` 는 `isEmpty` 와 관계없이 항상 필요한 `className` 이라는 의미를 담기 위해 아래로 수정하면 어떨까용
```suggestion
      <div className={`card-container ${isEmpty ? 'card-empty' : 'card-not-empty'}`}>
```